content-repo/extra-docs/articles/content-management.md

```diff
@@ -21,6 +73,59 @@ Consists of the following sections:
 
 ---
 
+## Building the CI/CD process
+
+### Recommneded Steps:
+1. Prepare the enviornment and the virtual enviornment to run demisto-sdk on.
+2. Create an ID set for the private repo, and merge it with Content repo's ID set.
```

Comment:
how? add the relevant command or a link to the step in the content build (link should use a specific commit)